QuoteEngine.py

```diff
@@ -0,0 +1,30 @@
+import requests
+import yaml
+import random
```

Comment:
You should arrange imports like this:

```
import random

import requests
import yaml
```

See http://legacy.python.org/dev/peps/pep-0008/#imports